### PR TITLE
MOD-6341:Build mariner2.0 docker image

### PR DESCRIPTION
**Pull request by oshadmi**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -125,13 +133,13 @@ workflows:
...
             parameters:
-              platform: [amzn2023, amzn2, rocky9, rocky8, centos7, lunar, jammy, focal, bionic, xenial, bookworm, bullseye, buster, fedora] # alpine3, alpine
+              platform: [mariner2.0, amzn2023, amzn2, rocky9, rocky8, centos7, lunar, jammy, focal, bionic, xenial, bookworm, bullseye, buster, fedora] # alpine3, alpine
           context: common
       - build-arm8:
           <<: *always
           matrix:
             parameters:
-              platform: [rocky9, rocky8, lunar, jammy, focal, bionic, bookworm, bullseye, buster, fedora] # alpine3, alpine
+              platform: [lunar, jammy, focal, bionic, bookworm, bullseye, buster, fedora] # rocky9, rocky8 # alpine3, alpine
```
Comment (adamiBs) [inline .circleci/config.yml:142]: Why did you need to comment out `rocky9, rocky8`?

Comment (oshadmi) [reply .circleci/config.yml:142]: It was [failing](https://app.circleci.com/pipelines/github/RedisLabsModules/redis-docker/515/workflows/425e8418-d6c6-4b5a-a68c-596ea88c69b4/jobs/8049?invite=true#step-105-10827_118) on arm

#24 [builder 12/19] RUN /build/deps/readies/bin/getpy3
#24 16.31 Raven's RPM repo for el8-aarch64 - base         1.8 MB/s | 402 kB     00:00    
#24 16.31 Raven's RPM repo for el8-aarch64 - modular pack 1.4 MB/s | 303 kB     00:00    
#24 16.31 Raven's RPM repo for el8-aarch64  - multimedia  1.3 kB/s | 146  B     00:00    
#24 16.31 Errors during downloading metadata for repository 'raven-multimedia':
#24 16.31   - Status code: 404 for https://pkgs.dyn.su/el8/multimedia/aarch64/repodata/repomd.xml (IP: 191.101.33.144)
#24 16.31 Error: Failed to download metadata for repo 'raven-multimedia': Cannot download repomd.xml: Cannot download repodata/repomd.xml: All mirrors were tried
#

**Review by adamiBs — APPROVED** [review]
